boards/st/nucleo_wb05kz/doc/index.rst

```diff
@@ -0,0 +1,152 @@
+.. _nucleo_wb05kz_board:
+
+ST Nucleo WB05KZ
+################
+
+Overview
+********
+
+The Nucleo WB05KZ board is a Bluetooth |reg| Low Energy wireless and ultra-low-power
+board featuring an ARM Cortex |reg|-M0+ based STM32WB05KZV MCU, embedding a
+powerful and ultra-low-power radio compliant with the Bluetooth® Low Energy
+SIG specification v5.4.
+
+.. image:: img/nucleo_wb05kz.jpg
+   :align: center
+   :alt: Nucleo WB05KZ
+
+More information about the board can be found on the `Nucleo WB05KZ webpage`_.
+
+Hardware
+********
+
+Nucleo WB05KZ provides the following hardware components:
+
+- STM32WB05KZV in VFQFPN32 package
+- ARM |reg| 32-bit Cortex |reg|-M0+ CPU
+- 64 MHz maximal CPU frequebct
+- 192 KB Flash
+- 24 KB SRAM
+
+More information about STM32WB05KZV can be found here:
+
+- `WB05KZ on www.st.com`_
+- `STM32WB05 reference manual`_
+
+
+Supported Features
+==================
+
+The Zephyr nucleo_wb05kz board configuration supports the following hardware features:
```

Comment:
put the board name as you use with west build, put \`\` around it and remove board configuration, it is board target